packages/functions_client/lib/src/functions_client.dart

```diff
@@ -71,11 +71,13 @@ class FunctionsClient {
     String functionName, {
     Map<String, String>? headers,
     Map<String, dynamic>? body,
+    Map<String, String>? queryParameters,
```

Comment:
From the [Uri constructor documentation](https://api.dart.dev/stable/3.3.4/dart-core/Uri/Uri.html)
> A value in the map must be either null, a string, or an [Iterable](https://api.dart.dev/stable/3.3.4/dart-core/Iterable-class.html) of strings. An iterable corresponds to multiple values for the same key, and an empty iterable or null corresponds to no value for the key.

So I think we could adapt the same type and use `Map<String,dynamic>` instead of `Map<String,String>` for the query parameters.

Reply:
Yes that's a good point, I'm updating this part right now.